update-affiliated/update_astropy_helpers.py

```diff
@@ -94,6 +94,11 @@ def open_pull_request(fork, repo):
 
     print(tmpdir)
 
+    report_user = '@astrofrog'
+
+    if username != 'astrofrog':
+        report_user = '@{} or @astrofrog'.format(username)
```

Comment:
That's fine by me, and once enough people understand how this script works we can drop this.